fakebook/karma.conf.js

```diff
@@ -18,6 +18,7 @@ module.exports = function (config) {
         // the possible options are listed at https://jasmine.github.io/api/edge/Configuration.html
         // for example, you can disable the random execution with `random: false`
         // or set a specific seed with `seed: 4321`
+        random: false
```

Comment:
I would take this out of the PR. We don't need to update the karma.conf.js on the branch.

Here is a link for that
https://stackoverflow.com/questions/39459467/remove-a-modified-file-from-pull-request